packages/devextreme/js/__internal/ui/splitter/splitter.ts

```diff
@@ -322,6 +325,18 @@ class Splitter extends (CollectionWidget as any) {
     });
   }
 
+  _updateNestedSplitterOption(optionName: string, optionValue: unknown): void {
+    this.option('items').forEach((item) => {
+      if (item?.splitter) {
+        const splitterItem = this._findItemElementByItem(item).children(`.${SPLITTER_CLASS}`);
```

Comment:
Splitter elements may not always be child items in wrappers. It's better to choose a solution that does not rely on the hierarchy of DOM elements.